src/components/BitcoinConnectElement.ts

```diff
@@ -52,4 +52,56 @@ export class BitcoinConnectElement extends InternalElement {
       this._modalOpen = currentState.modalOpen;
     });
   }
+
+  // Override updated method to set ARIA attributes based on state
+  protected override updated(
+    changedProperties: Map<string | number | symbol, unknown>
+  ): void {
+    super.updated(changedProperties);
+
+    // Set aria-live for connection status changes
+    if (this._connecting) {
+      this.setAttribute('aria-busy', 'true');
+      this.setAttribute('aria-live', 'polite');
+    } else {
+      this.removeAttribute('aria-busy');
+    }
+
+
+    // Announce errors if they exist
+    if (this._error) {
+      this.setAttribute('aria-live', 'assertive');
+      this.setAttribute('aria-label', `Error: ${this._error}`);
+    } else {
+      this.removeAttribute('aria-live');
+      this.removeAttribute('aria-label');
+    }
+
+    // Modal accessibility: handle modal state
+    if (this._modalOpen) {
+      this.setAttribute('aria-modal', 'true');
```

Comment:
Can these be directly set on the `bc-modal` component?

Reply:
They were removed with the other aria attributes to focus this pull request solely on keyboard navigation